core/src/main/resources/lib/hudson/projectViewRow.jelly

```diff
@@ -34,8 +34,5 @@ THE SOFTWARE.
     <j:forEach var="col" items="${columnExtensions}">
       <st:include page="column.jelly" it="${col}"/>
     </j:forEach>
-    <td>
```

Comment:
There was an extra row/column for the dashboard table causing unnecessary padding/spacing. Removing it doesn't seem to have broken anything - anybody know what it was there for?

Reply:
That's basically always been like that; the last part was a sortable column. Though it doesn't sort anything, likely used for spacing in past days, I guess?

Reply:
Looks like initially (pre-dating configurable columns) it was a spacer between the build icon and the data columns: https://github.com/jenkinsci/jenkins/blob/f7d7507f39de655c11fc28501ef3732304dc9a79/core/src/main/resources/lib/hudson/projectView.jelly#L118-L127

When https://github.com/jenkinsci/jenkins/commit/6f5dafd34320a89f1dfd1c48081eff4f6ee5a228 made the default columns part of the extension system, it was left in.